README.md

```diff
@@ -1,2 +1,4 @@
 # Bus-Mall
 A skymall-similar website for busses
+
+Pair programming with Emery, scaffolding our JS
```

Comment:
Going to want to start seeing the README files filled in with usage instructions and developer notes.